.github/workflows/on-prerelease.yml

```diff
@@ -107,39 +107,18 @@ jobs:
       actions: read
       contents: write
     steps:
-      - name: Securely retrieve the package from artifacts
-        uses: Ed-Fi-Alliance-OSS/slsa-github-generator/.github/actions/secure-download-artifact@main
-        with:
+     - name: Securely retrieve the package from artifacts
+       uses: Ed-Fi-Alliance-OSS/slsa-github-generator/.github/actions/secure-download-artifact@main
+       with:
           name: vscode-metaed.vsix
           path: vscode-metaed.vsix
           sha256: "${{ needs.package.outputs.hash-code }}"
-
-      - name: Attach to release
-        shell: pwsh
-        run: |
-          $release = "${{ github.ref_name }}"
-          $repo = "${{ github.repository }}"
-          $token = "${{ env.GITHUB_TOKEN }}"
-          $uploadName = "vscode-metaed.vsix"
-
-          $url = "https://api.github.com/repos/$repo/releases/tags/$release"
-
-          $gh_headers = @{
-              "Accept"        = "application/vnd.github+json"
-              "Authorization" = "Bearer $token"
-          }
-
-          $response = Invoke-RestMethod -Uri $url -Headers $gh_headers
-          $releaseId = $response.id
-
-          $url = "https://uploads.github.com/repos/$repo/releases/$releaseId/assets"
-
-          $gh_headers["Content-Type"] = "application/octet"
-          Invoke-RestMethod -Method POST `
-              -Uri "$($url)?name=$($uploadName)" `
-              -Headers $gh_headers `
-              -InFile $uploadName
-
+          
+     - name: Attach to release
+       uses: softprops/action-gh-release@c062e08bd532815e2082a85e87e3ef29c3e6d191 #v2.0.8
+       if: startsWith(github.ref, 'refs/tags/')
+       with:
+          files: vscode-metaed.vsix
```

Comment:
[Failing](https://github.com/Ed-Fi-Alliance-OSS/vscode-metaed-ide/actions/runs/10151951301/job/28072357830) with error:

```
Resource not accessible by personal access token
```


[According to this](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#update-a-release):

```
The fine-grained token must have the following permission set:

"Contents" repository permissions (write)
```

... but I see that this permission _is_ present ☹️ .

Reply:
https://github.com/softprops/action-gh-release?tab=readme-ov-file#permissions

> Note that if you intend to run workflows on the release event (on: { release: { types: [published] } }), you need to use a personal access token for this action, as the https://github.com/actions/create-release/issues/71.

At the top of this file I see that we have a token that is specifically for the purpose of writing to releases. Looks like you need to add:

```yml
        with:
          files: vscode-metaed.vsix
          token: ${{ env.GITHUB_TOKEN }}
```